nixos/modules/services/computing/slurm/slurm.nix

```
@@ -106,14 +132,13 @@ in
           '';
         };
 
-        extraConfig = mkOption {
-          type = types.lines;
-          default = "";
-          description = ''
-            Extra configuration for <literal>slurmdbd.conf</literal> See also:
-            <citerefentry><refentrytitle>slurmdbd.conf</refentrytitle>
-            <manvolnum>8</manvolnum></citerefentry>.
-          '';
+        settings = mkOption {
+          type = types.submodule {
+            freeformType = settingsFormat.type;
+
+            options = {};
+          };
```

```suggestion
          type = settingsFormat.type;
```

No need to use a freeform type when you don't have any options :)

---

fixed